.openshift/templates/ninja-board-back-end.yml

```diff
@@ -0,0 +1,228 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ninja-board-back-end
+  annotations:
+    iconClass: icon-jboss
+    tags: quarkus, java
+    description: Build the back-end Ninja Board App
+labels:
+  application: ${APPLICATION_NAME}
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: openjdk-11-rhel7
+  spec:
+    tags:
+    - from:
+        kind: DockerImage
+        name: ${DOCKER_IMAGE_REPOSITORY}
```

Comment:
This would need to be a specific tag, not a registry

Reply:
But when I import the image manually using this command:
`oc import-image --confirm openjdk/openjdk-11-rhel7 --from=registry.access.redhat.com/openjdk/openjdk-11-rhel7`

The actual resource that is generated has that name and kind.
Also, in the UI, the name it is referenced like this:
`registry/openjdk-11-rhel7:latest`

![image](https://user-images.githubusercontent.com/48076061/83800863-c5e53380-a66d-11ea-818c-533b1ba3b416.png)